WordPress/src/main/java/org/wordpress/android/ui/main/WPMainActivity.java

```diff
@@ -112,6 +113,9 @@ public void onCreate(Bundle savedInstanceState) {
         super.onCreate(savedInstanceState);
         setContentView(R.layout.main_activity);
 
+        // Allow vector drawables on pre-Lollipop (i.e. < API 21).
+        AppCompatDelegate.setCompatVectorFromResourcesEnabled(true);
```

Comment:
`AppCompatDelegate.setCompatVectorFromResourcesEnabled` is a static call, so once it's set, it will be used everywhere and since `WPMainActivity` is created just after the app starts, this won't make a difference to have it here on in the app creation. 

Well the only difference: having it in the app onCreate(), allows us to have vector selectors in the sign in/sign up screens (called before `WPMainActivity`) or in the Tab fragment (created during `WPMainActivity` creation).